samples/nrf9160/secure_services/README.rst

```diff
@@ -19,7 +19,7 @@ The following development board:
 
 * nRF9160 DK board (PCA10090)
 
-The following sample must be also flashed (happens automatically with the default configuration):
+The following sample must be also programmed (happens automatically with the default configuration):
```

Comment:
We should use the stock phrase here, as we do in the other nRF9160 samples: `* .. include:: /includes/spm.txt`

Reply:
Added.